pages.es/windows/test-netconnection.md

```diff
@@ -0,0 +1,13 @@
+# Test-NetConnection
+
+> Muestra información de diagnóstico de una conexión.
+> Este comando solo se puede utilizar a través de PowerShell.
+> Más información: <https://learn.microsoft.com/powershell/module/nettcpip/test-netconnection>.
+
+- Prueba una conexión y muestra resultados detallados:
+
+`Test-NetConnection -InformationLevel Detailed`
+
+- Prueba una conexión a un host remoto con un número de puerto específico:
```

Comment:
@kant @tricantivu should we translate host or not?

Reply:
I think not, some Spanish documentation [^1] uses the word "host". It could be a type of word which in Spanish is referred to as "extranjerismo".

[^1]: https://wiki.archlinux.org/title/Network_configuration_(Español)#Ping